package.json

```diff
@@ -80,9 +80,10 @@
     "test:node-fetch": "mocha --exit test/node-fetch",
     "test:fetch": "npm run build:node && borp --expose-gc --coverage -p \"test/fetch/*.js\" && borp --coverage -p \"test/webidl/*.js\"",
     "test:jest": "jest",
-    "test:tap": "tap test/*.js test/diagnostics-channel/*.js",
-    "test:node-test": "borp --coverage -p \"test/node-test/*.js\"",
-    "test:tdd": "tap test/*.js test/diagnostics-channel/*.js -w",
+    "test:tap": "tap test/*.js",
+    "test:node-test": "borp --coverage -p \"test/node-test/**/*.js\"",
+    "test:tdd": "tap test/*.js --coverage -w",
+    "test:tdd:node-test": "borp -p \"test/node-test/**/*.js\" -w",
```

Comment:
Had to split it as for some reason `diagnostics` tests were migrated but still run under `tap`, and failed only for this PR 🤔